devtools/maven/src/main/java/io/quarkus/maven/QuarkusBootstrapProvider.java

```diff
@@ -195,6 +171,96 @@ protected CuratedApplication curateApplication(QuarkusBootstrapMojo mojo) throws
             }
         }
 
+        protected AppArtifact managingProject(QuarkusBootstrapMojo mojo) {
+            if (mojo.appArtifactCoords() == null) {
+                return null;
```

Comment:
So if it's an external artifact, you return null here? What's bothering me a bit is that it wasn't set before so I'm not sure to understand why it's now set in the case that was already existing?

Reply:
Sure. When the appArtifact is configured, the current project is used as the environment for the build and tests. That's the meaning behind the managingProject. It's either null or the current project. Setting it to null is equivalent to not setting it at all.